Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/Extensions/ClaimSetResourceClaimActionAuthStrategiesExtension.cs

```diff
@@ -39,39 +39,39 @@ public static class ClaimSetResourceClaimActionAuthStrategiesExtension
     public static ClaimSetResourceClaimActionAuthStrategies?[] AddAuthorizationStrategyOverrides(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies,
         string actionName, AuthorizationStrategy? strategy)
     {
-        if (strategy != null)
+        if (strategy == null) return claimSetResourceClaimActionAuthStrategies;

```

Comment:
Before, this `!= null` check meant that everything below was indented. Easier to read by returning quickly here on the new line 42, and having one less indent below.